server/src/error-messages.js

```diff
@@ -1,5 +1,6 @@
 // This file includes error messages used in resolvers.
 module.exports = {
   userNotFoundMessage: 'User not found. You have to sign up first',
-  cannotCreateUserMessage: 'Fail to create the user. Please try again later'
-}
\ No newline at end of file
+  cannotCreateUserMessage: 'Fail to create the user. Please try again later',
```

Comment:
These 2 messages (cannotCreateUserMessage, notValidUserInfo) are very similar, and the logic that spits out these 2 messages aren't very different. Let's merge these 2 messages into one.